apps/src/lab2/views/dialogs/ShareDialog.tsx

```diff
@@ -28,17 +29,49 @@ const CopyToClipboardButton: React.FunctionComponent<{
   }, [shareUrl, projectType]);
 
   return (
-    <div>
-      <Button
-        iconLeft={{
-          iconName: copiedToClipboard ? 'clipboard-check' : 'clipboard',
-        }}
-        ariaLabel={i18n.copyLinkToProject()}
-        text={i18n.copyLinkToProject()}
+    <Button
+      iconLeft={{
+        iconName: copiedToClipboard ? 'clipboard-check' : 'clipboard',
+      }}
+      ariaLabel={i18n.copyLinkToProject()}
+      text={i18n.copyLinkToProject()}
+      type="secondary"
+      color="white"
+      size="m"
+      onClick={handleCopyToClipboard}
+      className={moduleStyles.copyLinkButton}
+    />
+  );
+};
+
+const AfeCareerTourBlock: React.FunctionComponent = () => {
+  const careersUrl =
+    'https://www.amazonfutureengineer.com/careertours/careervideos';
+
+  return (
+    <div className={classNames(moduleStyles.block, moduleStyles.blockAfe)}>
+      <Typography
+        semanticTag="h2"
+        visualAppearance="heading-md"
+        className={classNames(moduleStyles.heading, moduleStyles.blockHeading)}
+      >
+        {i18n.careerTourTitle()}
+      </Typography>
+      <img alt="" src="/shared/images/afe/afe-career-tours-0.jpg" />
```

Comment:
Alt text for this image?

Reply:
From reading a couple articles, it seems we don't need alt text if the nearby text conveys all the necessary information, which I think is the case here.